forge-game/src/main/java/forge/game/card/Card.java

```diff
@@ -4515,6 +4515,17 @@ public final Trigger getTriggerForStaticAbility(final String str, final StaticAb
         return result;
     }
 
+    public final Trigger addTriggerForStaticAbility(final Trigger trig, final StaticAbility stAb) {
```

Comment:
Probably should add Trigger Id as key too, in case of multiple Trigger with the same Trigger are exiled